docs/installingNative.md

```diff
@@ -92,6 +100,8 @@ Most of these prerequisites can be installed using the package manager of your O
 
     ```sh
     brew install git graphviz postgresql redis uv d2 node npm pgvector
+    # [optional] Some example questions need LaTeX
+    brew install --cask mactex
```

Comment:
Any reason to not use texlive here?

Reply:
Can you install `texlive` on a mac this way?

Reply:
https://www.tug.org/texlive/quickinstall.html

> For macOS (and MacOSX), we recommend installing [MacTeX](https://www.tug.org/mactex/), which has a native Mac installer, and contains all of TeX Live (and a few Mac-specific additions).

Reply:
Alternatively: https://formulae.brew.sh/formula/texlive

Reply:
![CleanShot 2025-05-06 at 18 19 43@2x](https://github.com/user-attachments/assets/176e9892-d270-4d63-b869-fe4a0eb19be6)
![CleanShot 2025-05-06 at 18 20 13@2x](https://github.com/user-attachments/assets/fa799cb0-de4b-453e-bf4d-043673fa4dc8)

I can recommend either, but mactex is the more popular option.

Reply:
```sh
curl -I https://ctan.math.illinois.edu/systems/mac/mactex/mactex-20250308.pkg | grep -i Content-Length
content-length: 6364206835

curl -I -H 'Authorization: Bearer QQ==' https://ghcr.io/v2/homebrew/core/texlive/blobs/sha256:dbb0b1a2a0b1d06263f2fe4e1b386cc2f682c4ab3a5d399b65cbfb3ce33ce4eb | grep -i Content-Length
content-length: 2082453817
```

`texlive` is less than a third the size of `mactex`. Based on that alone, I'd be inclined to go that route. `mactex` includes things like GUI applications that aren't relevant here.

Reply:
Changed in 1b086192ae93d397c504f8ac2cff96f8321b1637.